yt/frontends/boxlib/data_structures.py

```diff
@@ -1650,10 +1650,12 @@ def _parse_parameter_file(self):
                     self.parameters[l[0].strip()] = l[1].strip()
 
         # set the periodicity based on the integer BC runtime parameters
-        is_periodic = self.parameters["geometry.is_periodic"].split()
+        is_periodic = []
+        if 'geometry.is_periodic' is self.parameters:
```

Comment:
```suggestion
        if "geometry.is_periodic" in self.parameters:
```

Reply:
the `'` VS `"` stuff is just to satisfy `black`, but the `is -> in` looks like a serious typo (and it's what's causing flake8 errors)